src/herald/featureherald.js

```diff
@@ -60,6 +60,8 @@ olgm.herald.Feature.prototype.gmapFeature_ = null;
  */
 olgm.herald.Feature.prototype.label_ = null;
 
+olgm.herald.Feature.prototype.marker_ = null;
```

Comment:
Add comment block about this.  It's important to explain also what it does.  As I understand, instead of creating a style, we create a marker ?